File: src/test/java/net/kigawa/kutil/unit/dummy/Unit1.java
package net.kigawa.kutil.unit.dummy;

import net.kigawa.kutil.unit.annotation.*;
import net.kigawa.kutil.unit.api.component.*;
import net.kigawa.kutil.unit.dummy.parent.*;

@Kunit
public class Unit1 implements UnitOneToFourInterface
{
    protected final Unit4 unitDummy;
    protected final UnitContainer unitContainer;

    public Unit1(Unit4 unitDummy, UnitContainer unitContainer)
    {
        this.unitDummy = unitDummy;
        this.unitContainer = unitContainer;
    }
}

File: kunit/src/jvmTest/kotlin/net/kigawa/kutil/kunit/dummy/Unit1.kt
package net.kigawa.kutil.kunit.dummy

import net.kigawa.kutil.kunit.api.annotation.Kunit
import net.kigawa.kutil.kunit.api.component.container.UnitContainer
import net.kigawa.kutil.kunit.dummy.parent.UnitOneToFourInterface
import kotlin.test.Ignore

@Ignore
@Kunit
class Unit1(private val unitDummy: Unit4, private val unitContainer: UnitContainer) : UnitOneToFourInterface {
}
